sites/public/pages/create-account.tsx

```diff
@@ -38,10 +39,12 @@ export default () => {
 
   const onSubmit = async (data) => {
     try {
-      const { dob, ...rest } = data
+      const { dob, phoneNumber, ...rest } = data
       await createUser({
         ...rest,
         dob: moment(`${dob.birthYear}-${dob.birthMonth}-${dob.birthDay}`),
+        // Convert (123) 456-7890 to E.164 format with US country code: +11234567890
+        phoneNumber: phoneNumber.replace(/\((\d{3})\) (\d{3})-(\d{4})/, "+1$1$2$3"),
```

Comment:
Does the input component guarantee the phone number will always be in this format?

Reply:
From what I could tell, yes.